frontend/src/components/config-details.ts

```diff
@@ -96,6 +96,28 @@ export class ConfigDetails extends LiteElement {
       }
     };
 
+    const renderSize = (valueBytes?: number | null, fallbackValue?: number) => {
```

Comment:
We use the shoelace [`sl-format-bytes`](https://shoelace.style/components/format-bytes/) component to render bytes, should work in this case, too!

```ts
<sl-format-bytes
    value=${valueBytes}
    display="narrow"
></sl-format-bytes>
```

Reply:
I tried using this but `sl-format-bytes` uses 1000 bytes/kb instead of 1024 bytes/kb, which results in the config detail having an inaccurate number (e.g. "3.22 GB" instead of "3 GB"). It might be better to keep it as-is but use the same formatting as `<sl-format-bytes display="narrow">` - what do you think?

Reply:
Or we could make a custom `<btrix-format-bytes>` that calculates with 1024 bytes/kb?

Reply:
Got it! It looks like `sl-format-bytes` uses [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#syntax) under the hood, so to format it consistently for different locales you could do something like:
```ts
const value = 1000; // example
const formatter = new Intl.NumberFormat(undefined, {
  style: "unit",
  unit: "gigabyte",
  unitDisplay: "narrow"
});

formatter.format(1000); // outputs "1,000GB"
```